### PR TITLE
Updated to use systemd on Fedora's RPM

### PR DESCRIPTION
**Pull request by kjdev**

### CODE REVIEW

File: pkg/rpm/unit.spec.in [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # distribution specific definitions
-%define use_systemd (0%{?rhel} && 0%{?rhel} >= 7) || (0%{?suse_version} >= 1315)
+%define use_systemd (0%{?rhel} && 0%{?rhel} >= 7) || (0%{?fedora} && 0%{?fedora} >= 18) || (0%{?suse_version} >= 1315)
```
Comment (defanator) [inline pkg/rpm/unit.spec.in]: I believe `0%{?fedora} >= 18` is enough here. Also, I'd place this after `suse_version` check just to honor original ordering of platforms as they've been added to unit spec.

File: pkg/rpm/unit.spec.in [hunk 2]
```diff
@@ -15,6 +15,11 @@ BuildRequires: systemd-units
 %endif
 %endif
 
+%if 0%{?fedora} >= 18
```
Comment (defanator) [inline pkg/rpm/unit.spec.in]: Ditto (ordering) - I'd place this one after SUSE block.

**Comment by kjdev** [comment]
Fixed and updated.

**Comment by remicollet** [comment]
+1 to fix Fedora

Isn't better to simple use the 

```
%if %{use_systemd}
BuildRequires:     systemd
Requires(post):    systemd
Requires(preun):   systemd
Requires(postun):  systemd
%endif


```Notice: this will also fix the EL8 case (current condition is `%if 0%{?rhel} == 7` which is obviously wrong)

**Comment by remicollet** [comment]
Also better to use %{?systemd_requires} (when defined)

**Comment by remicollet** [comment]
I include another way to fix this in pr #212

**Comment by komapa** [comment]
Remi, do you plan to have nginx unit as part of your PHP rpm repos?

**Comment by remicollet** [comment]
> Remi, do you plan to have nginx unit as part of your PHP rpm repos?

unit-php module are already there https://blog.remirepo.net/post/2019/01/14/PHP-with-the-NGINX-unit-application-server

I will probably includes nginx packages (which will include changes from pr #212 ), at least for Fedora and RHEL-8_Beta

**Comment by remicollet** [comment]
@komapa "unit" packages now available in my repository (Fedora 26 to 29), only with unit-php (no other languages planed)

**Pull request closed without merging**